code/datalab-app/src/app/components/app/Navigation.js

```diff
@@ -36,10 +36,10 @@ const styles = theme => ({
   },
 });
 
-const Navigation = ({ classes, children }) => (
+const Navigation = ({ classes, children, identity }) => (
   <div className={classes.container}>
     <div className={classes.appFrame}>
-      <TopBar/>
+      <TopBar identity={identity} />
```

Comment:
Surprised this didn't break the spec.

Reply:
I did update the snapshot image. Will add some additional unit test to cover the new components.